source/installation-guide/virtual-machine.rst

```diff
@@ -44,3 +44,7 @@ Wazuh provides a pre-built virtual machine image (OVA) that you can directly imp
 .. note::
 
         You can also download the virtual machine using the OVF format. You need to download the `OVF <https://packages.wazuh.com/vm/wazuh3.6.1_6.4.0.ovf>`_ file and the `vmdk <https://packages.wazuh.com/vm/wazuh3.6.1_6.4.0-disk001.vmdk>`_ disk.
+        
+.. note::
+
+        If you need to update your OVA virtual machine, you can consult the article `Upgrade to latest version of Wazuh 3.x <https://documentation.wazuh.com/current/installation-guide/upgrading/latest_wazuh3_minor.html>`_ also we recommend to update repository with ``yum update`` command.
```

Comment:
The end of the sentence doesn't have many sense:
![image](https://user-images.githubusercontent.com/10928321/46466776-74b3db00-c7cc-11e8-85ba-b2f199e0ba90.png)

I would say:
```
"... you can check out :ref:`this article <upgrading_latest_minor>`. We also recommend updating your repositories using the ``yum update`` command."
```